spec/system/comparisons/hot_water_efficiency_spec.rb

```diff
@@ -59,4 +60,24 @@
 
     it_behaves_like 'a school comparison report with a chart'
   end
+
+  context 'when there are schools with swimming pools' do
+    let!(:other_school) { create(:school, has_swimming_pool: true) }
+
+    before do
+      create(:alert, school: other_school, alert_generation_run: alert_run, alert_type: alert_type, variables: variables)
+      visit "/comparisons/#{key}"
+    end
+
+    it 'still has a single school' do
+      within("##{report.key}-table") do
+        expect(page).to have_link(school.name)
+        expect(page).not_to have_link(other_school.name)
+      end
+    end
+
+    it 'has custom unlisted message' do
```

Comment:
Is it worth testing the different output in the modal too i.e listing if the school has a swimming pool?

Reply:
Yes. Although struggling to get the test to actually open the modal at the moment. Following same pattern as in compare_spec.rb but any assertions fail to find `.modal-body`

Reply:
Have you got js: true switched on?